balances/src/tests.rs

```diff
@@ -24,10 +24,9 @@ use encointer_primitives::{
 };
 use frame_support::{assert_noop, assert_ok, traits::OnInitialize};
 use mock::{new_test_ext, EncointerBalances, System, TestRuntime};
-use test_utils::{
-	helpers::{last_event, register_test_community},
-	AccountKeyring,
-};
+use test_utils::{helpers::last_event, AccountKeyring};
+
+use sp_std::str::FromStr;
```

Comment:
We want to prevent empty lines in the import sections because this screws up the formatter.